app/controllers/customer_reviews_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CustomerReviewsController < ApplicationController
+
+    def new
```

Comment:
I think this should be named company_review, do you think that makes more sense?